sbt-dotty/sbt-test/source-dependencies/export-clauses/build.sbt

```diff
@@ -0,0 +1,12 @@
+import complete.DefaultParsers._
+
+val checkIterations = inputKey[Unit]("Verifies the accumlated number of iterations of incremental compilation.")
+
+checkIterations := {
+  val analysis = (compile in Compile).value.asInstanceOf[sbt.internal.inc.Analysis]
+
+  val expected: Int = (Space ~> NatBasic).parsed
+  val actual: Int = analysis.compilations.allCompilations.size
+  assert(expected == actual, s"Expected $expected compilations, got $actual")
+}
```

Comment:
This will no longer work now that https://github.com/lampepfl/dotty/pull/10498 has been merged because sbt 1.4 uses batch mode for scripted tests. This test needs to be updated to use recordPreviousIterations / checkIterations like was done for the existing tests in https://github.com/lampepfl/dotty/pull/10498/commits/1a3100e3f85c67780a268dd7ad7584e53ff27db6

Reply:
thank you, I've made the change